client/signup/steps/survey/survey-step-v1.jsx

```diff
@@ -98,25 +98,47 @@ export default React.createClass( {
 					</div>
 
 					<Card className={ subVerticalsClasses }>
-						<BackButton isCompact className="survey-step__title" onClick={ this.showStepOne }>{ this.state.stepOne && this.state.stepOne.label }</BackButton>
+						<BackButton isCompact className="survey-step__title" onClick={ this.showStepOne }>
+							{ this.state.stepOne && this.state.stepOne.label }
+						</BackButton>
 						{ this.state.stepTwo.map( this.renderStepTwoVertical ) }
 					</Card>
 				</div>
 			</div>
 		);
 	},
 
+	renderHeaderText() {
+		if ( this.props.surveySiteType === 'blog' ) {
+			const { domain } = this.props.queryObject;
```

Comment:
`queryObject` is not defined if no query parameters are passed, so [this breaks the `blog` and `delta-blog` flows](https://cloudup.com/cHBEN0zUcHq).

We can fix it with a presence check for `queryObject`.